blockstore/blockstore.go

```diff
@@ -310,6 +317,227 @@ func (bs *blockstore) AllKeysChan(ctx context.Context) (<-chan cid.Cid, error) {
 	return output, nil
 }
 
+// GetManyOption is a getManyBlockStore option implementation
+type GetManyOption struct {
+	f func(bs *getManyBlockStore)
+}
+
+// NewGetManyBlockstore returns a default GetManyBlockstore implementation
+// using the provided datastore.TxnDatastore backend.
+func NewGetManyBlockstore(d ds.TxnDatastore, opts ...GetManyOption) GetManyBlockstore {
```

Comment:
You implement both GetMany and PutMany but only return an interface which only expose GetMany.

Reply:
Ahh thanks, good catch. This is a bit of testament to how much the `PutMany` is an afterthought, as we are actually only using the `GetMany` method currently. Fixing now.

Reply:
Oh actually `PutMany` is already a method of the regular `Blockstore` interface so it is a part of the `GetManyBlockstore` interface returned here since it extends `Blockstore`. But, this can be made much more clear by declaring it explicitly in the new interface and also by adjusting the name of this interface and its implementation to reflect the fact that the `PutMany` method operates differently (through `ds.TxnFeature` instead of `ds.Batching`) and that the new `GetMany` method isn't the only difference.

Reply:
Proposing to change the name to `TxnBlockstore`

Reply:
:facepalm: mb, didn't knew we already had `PutMany`, `GetManyBlockstore` is fine.